tests/test_main.py

```diff
@@ -5,7 +5,7 @@
 import unittest
 from datetime import date
 
-from simple_database import create_database, connect_database
+from simple_database.dbmodule import create_database, connect_database
```

Comment:
if you moved your imports into the main.py, you would not have to alter the tests.